app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -34,30 +34,34 @@
       <%= tag.span error, id: "#{id}-error", class: "govuk-error-message" %>
     <% end %>
 
-    <%= tag.div class: "govuk-checkboxes", data: {
+    <%= tag.ul class: "govuk-checkboxes gem-c-checkboxes__list", data: {
```

Comment:
Is the class `govuk-checkboxes` still in use? Can't see it in the sass and turning it off doesn't seem to make any difference.

Reply:
hey yeah - it's needed by the design systems https://design-system.service.gov.uk/components/checkboxes/

Reply:
Interesting - might be an unused class in the Design system, same thing happens there (the class appears to have no styles associated with it and removing it has no effect). Fair enough though - best leave it.